pkg/kmodule/kmodule_linux.go

```diff
@@ -124,6 +129,32 @@ func ProbeOptions(name, modParams string, opts ProbeOpts) error {
 	return nil
 }
 
+func checkBuiltin(moduleDir string, deps depMap) error {
+	f, err := os.Open(filepath.Join(moduleDir, "modules.builtin"))
+	if os.IsNotExist(err) {
+		return nil
+	} else if err != nil {
+		return fmt.Errorf("could not open builtin file: %v", err)
+	}
+	defer f.Close()
+
+	scanner := bufio.NewScanner(f)
+	for scanner.Scan() {
+		txt := scanner.Text()
+		modPath := filepath.Join(moduleDir, strings.TrimSpace(txt))
+		if deps[modPath] == nil {
+			deps[modPath] = new(dependency)
+		}
+		deps[modPath].state = builtin
+	}
+
+	if err := scanner.Err(); err != nil {
```

Comment:
Could be simplified to just 

```go
return scanner.Err()
```

Reply:
Thanks for the feedback. I've pushed a new version with this change.